pkg/controllers/manifestwork.go

```diff
@@ -69,11 +70,34 @@ func ScaffoldManifestwork(hyd *hypdeployment.HypershiftDeployment) (*workv1.Mani
 					hyd.GetName()),
 			},
 		},
-		Spec: workv1.ManifestWorkSpec{},
-	}
-
-	if hyd.Spec.Override == hypdeployment.InfraOverrideDestroy {
```

Comment:
I deleted the code:
```
	if hyd.Spec.Override == hypdeployment.InfraOverrideDestroy {
		w.Spec.DeleteOption = &workv1.DeleteOption{PropagationPolicy: workv1.DeletePropagationPolicyTypeOrphan}
	}
```
because I think this is dead code. if the `hyd.Spec.Override == hypdeployment.InfraOverrideDestroy`, the controller will not use manifestwork, right?
@ianzhang366 Please help to check.